platformio_override.sample.ini

```diff
@@ -533,6 +533,7 @@ monitor_filters = esp32_exception_decoder
 # 433MHz RF remote example for esp32dev
 [env:esp32dev_usermod_RF433]
 extends = env:esp32dev
-build_flags = ${env:esp32dev.build_flags} -D USERMOD_RF433
+build_flags = ${env:esp32dev.build_flags}
+custom_usermods = usermod_v2_RF433
 lib_deps = ${env:esp32dev.lib_deps}
```

Comment:
The entire `lib_deps` statement should be deleted too. That's handled automatically by the PlatformIO Libraries feature we use now.

In this case it's handled here now:

https://github.com/wled/WLED/blob/main/usermods/usermod_v2_RF433/library.json